src/main/java/com/kryptnostic/datastore/cassandra/CommonColumns.java

```diff
@@ -30,19 +30,22 @@
 
 public enum CommonColumns implements ColumnDef {
     ACLID( DataType.uuid() ),
+    AUDIT_ID( DataType.uuid() ),
```

Comment:
@geekbeast you said audit ID is just a vertex ID. Should `AUDIT_ID` be a `UUID`?